hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/helpers/OmBucketInfo.java

```diff
@@ -61,14 +63,20 @@
    */
   private StorageType storageType;
   /**
-   * Creation time of bucket.
+   * Bucket encryption key info if encryption is enabled.
    */
-  private final long creationTime;
+  private BucketEncryptionKeyInfo bekInfo;
 
   /**
-   * Bucket encryption key info if encryption is enabled.
+   * Bucket Flags that define the behavior of a bucket.
+   *
+   * enum BucketFlags {
+   *     CONFUSING_CHARS = 1;
```

Comment:
Purpose of this? And how this will be used?
I don't see the usage of it anywhere in the actual bucket logic/ client setting these flags.

Reply:
It is to enforce character sets which will not conflict with the S3, O3 and other file protocols. It is not being used now

Reply:
Sorry, I have not understood, what is meant by not conflict with the S3, O3, and other file protocols. 
Any example to help me understand here?